More unit tests to IPv6 fragmentation reassembly

rveerama1 opened the pull request.

rveerama1 committed:
tests: net: Add more tests to ipv6 fragmentation

IPv6 fragmentation test case for reassembly.

Signed-off-by: Ravi kumar Veeramally <ravikumar.veeramally@linux.intel.com>

rveerama1 committed:
net: ipv6: Trivial changes in fragmentation debug

Values are misleading when debugging IPv6 fragmentation.

Signed-off-by: Ravi kumar Veeramally <ravikumar.veeramally@linux.intel.com>

tbursztyka reviewed (changes requested):
Remove the DBG/NET_DBG change (or split it in another patch)

rveerama1 committed:
net: tests: ipv6_fragment: Use NET_DBG instead of DBG

ipv6_fragment tests registered log module but still use DBG
macro of printk. Replaced them with NET_DBG.

Signed-off-by: Ravi kumar Veeramally <ravikumar.veeramally@linux.intel.com>

jukkar reviewed (approved):
LGTM

rveerama1 commented:
> Remove the DBG/NET_DBG change (or split it in another patch)

Done.

galak commented:
If you desire this fix in 1.14, please open or associate this PR with an Issue.

jukkar commented:
@rveerama1 please rebase against latest master and re-submit.

rveerama1 commented:
@jukkar done.

tbursztyka commented on tests/net/ipv6_fragment/src/main.c:
```diff
@@ -1055,28 +1050,28 @@ static int verify_fragment(struct net_pkt *pkt)
 		if (offset != pkt_recv_data_len) {
-			DBG("Invalid offset %d received %d\n",
+			NET_DBG("Invalid offset %d received %d",
 			    offset, pkt_recv_data_len);
```
seems to be badly indented here

tbursztyka commented on tests/net/ipv6_fragment/src/main.c:
```diff
@@ -1094,14 +1089,14 @@ static int verify_fragment(struct net_pkt *pkt)
 		if (pkt_data_len != pkt_recv_data_len) {
-			DBG("Invalid amount of data received (%d vs %d)\n",
+			NET_DBG("Invalid amount of data received (%d vs %d)",
 			    pkt_data_len, pkt_recv_data_len);
```
same

rveerama1 commented:
@tbursztyka done.

rveerama1 commented:
@tbursztyka Take a look this one also.
